io.sloeber.product/p2.inf

```diff
@@ -0,0 +1,35 @@
+requires.1.namespace=org.eclipse.equinox.p2.iu
+requires.1.name=sloeber.configuration.win32
+requires.1.filter=(osgi.os=win32)
+requires.1.range=[1.0.0,1.0.0]
+requires.1.greedy=true
+
+requires.2.namespace=org.eclipse.equinox.p2.iu
+requires.2.name=sloeber.configuration.default
+requires.2.filter=(!(osgi.os=win32))
+requires.2.range=[1.0.0,1.0.0]
+requires.2.greedy=true
+
+
+units.1.id=sloeber.configuration.win32
+units.1.version=1.0.0
+units.1.provides.1.namespace=org.eclipse.equinox.p2.iu
+units.1.provides.1.name=sloeber.configuration.win32
+units.1.provides.1.version=1.0.0
+units.1.filter=(osgi.os=win32)
+units.1.touchpoint.id=org.eclipse.equinox.p2.osgi
+units.1.touchpoint.version=1.0.0
+units.1.instructions.configure=setProgramProperty(propName:osgi.instance.area.default,propValue:@user.home/My Documents/sloeber-workspace);
+units.1.instructions.unconfigure=setProgramProperty(propName:osgi.instance.area.default,propValue:);
+
+units.2.id=sloeber.configuration.default
+units.2.version=1.0.0
+units.2.provides.1.namespace=org.eclipse.equinox.p2.iu
+units.2.provides.1.name=sloeber.configuration.default
+units.2.provides.1.version=1.0.0
+units.2.filter=(!(osgi.os=win32))
+units.2.touchpoint.id=org.eclipse.equinox.p2.osgi
+units.2.touchpoint.version=1.0.0
+units.2.instructions.configure=setProgramProperty(propName:osgi.instance.area.default,propValue:@user.home/Documents/sloeber-workspace);
+units.2.instructions.unconfigure=setProgramProperty(propName:osgi.instance.area.default,propValue:);
+
```

Comment:
https://wiki.eclipse.org/Equinox/p2/Customizing_Metadata
and copied from http://bit.ly/2j04DvN
Makes the build create extra parameters in config.ini that define default workspace location